docs/app/references/configuration.mdx

```diff
@@ -588,6 +589,55 @@ The `**/node_modules/**` pattern is automatically added to `excludeSpecPattern`
 and does not need to be specified (and can't be overridden). See [`e2e`](#e2e)
 or [`component`](#component) testing-specific options.
 
+### injectDocumentDomain
+
+This option is <Badge type="caution">deprecated</Badge>, and will be removed in the next
+major version of Cypress.
+
+Set this configuration option is `true` to instruct Cypress to
```

Comment:
```suggestion
Set this configuration option to `true` to instruct Cypress to
```

Reply:
addressed in https://github.com/cypress-io/cypress-documentation/pull/5974/commits/644aa612b9a136472c6f1be955f49458f3770b70